### PR TITLE
[#772] Add Airy Core, Airy Enterprise, Need help? and github links to navigationBar

### PR DESCRIPTION
**Pull request by thorstenairy**

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -27,6 +27,32 @@ module.exports = {
...
+          href: 'https://github.com/airyhq/airy',
+          className: 'header-github-link',
+          'aria-label': 'GitHub repository',
+          position: 'right',
+        },
+        {
+          target: '_self',
+          label: 'Need help?',
+          position: 'right',
+          href: 'https://airy.co/docs/getting-started/troubleshooting',
```
Comment (lucapette) [inline docs/docusaurus.config.js:53]: this links needs to be relative to the project

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -27,6 +27,32 @@ module.exports = {
...
+          target: '_self',
+          label: 'Airy Core',
+          position: 'left',
+          href: 'https://airy.co/docs/airycore',
+        },
+        {
+          target: '_self',
+          label: 'Airy Enterprise',
+          position: 'left',
+          href: 'https://airy.co/docs/airyenterprise',
```
Comment (lucapette) [inline docs/docusaurus.config.js:41]: this doesn't exist yet I think @sgarroum wrote the issue with the future in mind... :)

https://enterprise.airy.co

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -27,6 +27,32 @@ module.exports = {
         alt: 'Airy Documentation',
         src: 'img/logo.svg',
       },
+      items: [
+        {
+          target: '_self',
+          label: 'Airy Core',
+          position: 'left',
+          href: 'https://airy.co/docs/airycore',
```
Comment (lucapette) [inline docs/docusaurus.config.js:35]: also here

https://docs.airy.co

File: docs/docusaurus.config.js (hunk 2 above)
Comment (thorstenairy) [reply docs/docusaurus.config.js:41]: perfect, was just writing to him 👍

**Review by lucapette — CHANGES_REQUESTED** [review]
Menu looks great but the links are "wrong" (sorry :))

**Review by AitorAlgorta — APPROVED** [review]

**Review by AitorAlgorta — APPROVED** [review]

**Review by lucapette — APPROVED** [review]
:shipit:

**Pull request merged**